sale_configurator_option/models/sale.py

```diff
@@ -44,6 +68,23 @@ class SaleOrderLine(models.Model):
         ondelete="set null",
         compute="_compute_product_option_id",
     )
+    origin_copy_id = fields.Many2one(
```

Comment:
We should not add a new field for this.
We should find an other implementation

Reply:
Where is -the problem ? In odoo there are many technical fields

Reply:
ping @sebastienbeau

Reply:
I suggest to add this solution until we found an other one (we can improve later). there is a real bug now.

Reply:
ping @sebastienbeau